pkg/bisect/bisect.go

```diff
@@ -19,15 +19,16 @@ import (
 )
 
 type Config struct {
-	Trace     debugtracer.DebugTracer
-	Fix       bool
-	BinDir    string
-	Ccache    string
-	Timeout   time.Duration
-	Kernel    KernelConfig
-	Syzkaller SyzkallerConfig
-	Repro     ReproConfig
-	Manager   *mgrconfig.Config
+	Trace          debugtracer.DebugTracer
+	Fix            bool
+	BisectCompiler string
```

Comment:
Nit: the `Bisect` prefix here is not necessary, everything is related to bisection anyway.

Reply:
Also, maybe this field better belongs to the `KernelConfig`? Though I'm not 100% sure either.

Reply:
I prefixed it Bisect, as one of my next changes will be to also plumb the regular compiler through this config. This was one change suggested by Dmitry in order to allow for at least some bisection of recently introduced issues for those setup where we have specialised compilers with features not yet upstreamed to clang/gcc. IIRC this is currently the case for some sanitizers.

Reply:
Re moving to KernelConfig - I also considered this, but wasn't sure either. Let me know what you prefer and I'll do it. This is an internal struct anyhow, so we can also change this later if we don't like it w/o disturbing users.